routes/socket/user-events.js

```diff
@@ -324,25 +324,10 @@ const handleUserLeaveGame = (socket, game, data, passport) => {
 	if (playerIndex > -1) {
 		if (game.publicPlayersState[playerIndex].isRemakeVoting) {
 			// Count leaving the game as rescinded remake vote.
-			const minimumRemakeVoteCount = (() => {
-				switch (game.general.playerCount) {
-					case 5:
-						return 4;
-					case 6:
-						return 5;
-					case 7:
-						return 5;
-					case 8:
-						return 6;
-					case 9:
-						return 6;
-					case 10:
-						return 7;
-				}
-			})();
+			const minimumRemakeVoteCount = (game.general.playerCount - game.customGameSettings.fascistCount);

```

Comment:
This is such a good change it makes me wonder why we didn't do this in the first place.

Reply:
ah... actually there is a problem here. `customGameSettings` is a `gameSummary` property, not a `game` property. `handleUserLeaveGame` is being passed a raw `game` object. This will probably error. Please find another way to do this.

Reply:
im quite new with this code so i dont know exactly how all things are connected so i doubt i can make it, but i will try